### PR TITLE
Create CODE_OF_CONDUCT.md

### LINKED ISSUE #451: Add a code of conduct for the repository.
#### This issue is about [Algo/DS Name](link to resource for the Algo/DS).

- [ ] Issue Title is in form `Algo/DS Name [Language]`.
- [ ] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [ ] This Algo/DS is worth adding to this repository.
- [ ] Assign this issue to me (I want to work on this).
	- [ ] I ensure that I am not already assigned to 2 or more issues.
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

### PR DESCRIPTION
**Pull request by Monal5031**
Fixes #451 

#### By submitting this pull request I confirm I've read and complied with the below declarations.

- [x] I have read the [Contribution guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md) and I am confident that my PR reflects them.
- [x] I have followed the [coding guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#cs) for this project.
- [ ] My code follows the [skeleton code structure](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#sample).
- [ ] This pull request has a descriptive title. For example, `Added {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.
- [x] This pull request will be closed if I fail to update it even once in a continuous time span of 7 days.

### CODE REVIEW

**Comment by singhpratyush** [comment]
Hi! Thanks for showing interest in contributing to this repository.

Please make sure that you have ticked the points mentioned in PR description correctly.

Thanks again!

**Comment by Monal5031** [comment]
@singhpratyush email id used: `shadi.monty@gmail.com`

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at shadi.monty@gmail.com. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```
Comment (aashutoshrathi) [inline CODE_OF_CONDUCT.md]: Please change `shadi.monty@gmail.com` to something more human.

Comment (aashutoshrathi) [reply CODE_OF_CONDUCT.md]: @Monal5031 better change it to your iiitv's email.

Comment (Monal5031) [reply CODE_OF_CONDUCT.md]: Yeah will do it, once we get some time from those projects 😑 and "VIDEOS"

**Comment by Monal5031** [comment]
@singhpratyush @aashutoshrathi Update the email please review.

**Pull request merged**